src/middlewared/middlewared/utils/license.py

```diff
@@ -10,4 +10,5 @@
     9: "ES102",
     10: "ES102G2",
     11: "ES60G2",
+    12: "ES24N",
```

Comment:
I'm assuming this has not been added to the license server, yet? This key in this dict matches the unique id for the database table on an internal license server that we have. I don't expect you to know that since it's tribal knowledge.

Reply:
Correct.  I have zero knowledge wrt the license server, so the above may need to be changed.